test/parallel/test-buffer-concat.js

```diff
@@ -98,3 +100,24 @@ assert.deepStrictEqual(
 assert.deepStrictEqual(Buffer.concat([new Uint8Array([0x41, 0x42]),
                                       new Uint8Array([0x43, 0x44])]),
                        Buffer.from('ABCD'));
+            
+test("concat buffer greater than 4GB in length", () => {
```

Comment:
This file doesn't make use of the test module, IMO it's better just to add the code without the test module

Reply:
Resolved by [2f1b905](https://github.com/nodejs/node/pull/55492/commits/2f1b905f005f316ec7193b2fc96680bdbc46e5e0)